src/proxies/panel_manager.js

```diff
@@ -76,6 +77,16 @@ PanelManager.openFavorite = async function () {
   })
 }
 
+PanelManager.openCategory = async function (options) {
+  __panel_manager.panels.forEach((panel) => {
```

Comment:
Is the `__panel_manager` comme from `window` ? If so, explicit it to avoid misunderstanding of the code

Reply:
Used as is everywhere